### ISSUE TITLE
Narrator does not announce the tooltip details which is appearing on “Title” edit field.

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#131]
User Experience:

If screen reader does not announce tooltip details then the people with  disabilities and cognitive limitations will not be access the instructions for using the content.

Note: User credentials should NOT be included in the bug.

 

Repro Steps:

1. Open Azure portal with valid credentials and create a API Management.

2. Click Developer Portal button to navigate to the portal.

3. Wait portal to load in design mode

4. Navigate to the Left nav and select Pages button.

5. Navigate to the Pages window. Select "APIs: List" option.

6. Navigate to the any editable control details and select Add widget.

7. Navigate to the various controls of the "Add widget" dialog box.

8. Enable screen reader and navigate to “Title” Edit field.

9. Verify the issue.

 

Actual Result:

Narrator does not announce the tooltip details as ‘This fields is required’ which is appearing on “Title” Edit field.

 

 

Expected result:

Narrator should announce the tooltip details as ‘This fields is required’ which is appearing on “Title” Edit field.

 

Note:

This issue does not exist with JAWS and NVDA.

 

MAS Reference: 

MAS 4.2.1 - Object Information (33A)

 

Reference Links

Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
Please reach out to C&AI Teams channel for any process related queries
